app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError. "
+                                     "No vaccination")
+        elif visitor.get("vaccine").get("expiration_date") \
```

Comment:
Now you have 2 possible errors:
visitor.get("vaccine") will return None
visitor.get("vaccine").get("expiration_date") will return None
In this case, you can use it as [] no need .get here